providers/transip/auditrecords.go

```diff
@@ -11,19 +11,21 @@ import (
 func AuditRecords(records []*models.RecordConfig) []error {
 	a := rejectif.Auditor{}
 
+	a.Add("ALIAS", rejectif.LabelNotApex) // Last verified 2024-01-11
+
 	a.Add("MX", rejectif.MxNull) // Last verified 2023-12-04
 
-	a.Add("TXT", rejectif.TxtHasBackticks) // Last verified 2023-12-04
+	a.Add("TXT", rejectif.TxtHasBackticks) // Last verified 2024-01-11
 
-	a.Add("TXT", rejectif.TxtHasDoubleQuotes) // Last verified 2023-12-04
+	a.Add("TXT", rejectif.TxtHasBackslash) // Last verified 2024-01-11
 
-	a.Add("TXT", rejectif.TxtHasBackslash) // Last verified 2023-12-04
+	a.Add("TXT", rejectif.TxtStartsOrEndsWithSpaces) // Last verified 2024-01-11
 
-	a.Add("TXT", rejectif.TxtStartsOrEndsWithSpaces) // Last verified 2023-12-10
+	a.Add("TXT", rejectif.TxtIsEmpty) // Last verified 2024-01-11
 
-	a.Add("TXT", rejectif.TxtIsEmpty) // Last verified 2023-12-10
+	a.Add("TXT", rejectif.TxtLongerThan(1024)) // Last verified 2024-01-11
 
-	a.Add("TXT", rejectif.TxtLongerThan(1024)) // Last verified 2023-12-15
+	a.Add("TXT", rejectif.TxtHasTrailingSpace) // Last verified 2024-01-11
```

Comment:
Please keep in sorted order:

```
  a.Add("ALIAS", rejectif.LabelNotApex) // Last verified 2024-01-11 

  a.Add("MX", rejectif.MxNull) // Last verified 2023-12-04

  a.Add("TXT", rejectif.TxtHasBackslash) // Last verified 2024-01-11

  a.Add("TXT", rejectif.TxtHasBackticks) // Last verified 2024-01-11

  a.Add("TXT", rejectif.TxtHasTrailingSpace) // Last verified 2024-01-11

  a.Add("TXT", rejectif.TxtIsEmpty) // Last verified 2024-01-11

  a.Add("TXT", rejectif.TxtLongerThan(1024)) // Last verified 2024-01-11

  a.Add("TXT", rejectif.TxtStartsOrEndsWithSpaces) // Last verified 2024-01-11
```

Reply:
I have not adjusted any ordering. That's the git diff being misleading. Please check commit by commit. ☺️

Reply:
Duh. I'm illiterate.